docs/resources/onboarding/docker.md

```diff
@@ -87,6 +105,30 @@ If you've got Docker running, you should be able to see your database in the web
 
 ### 2a) One time setup
 
+Here again there is the option to run python inside the `docker-compose` environment or using your local python setup.  The easiest method is to run using the first option below, but the others will work as well.
+
+**Full Docker Compose (docker Python)**
+
+1)  `docker-compose ps` to see the containers that are running.  One of them should be **housinginsights_sandbox_1**.  This is the container that is running the miniconda3 python environment.  There is no need to activate or deactivate an virtualenv since it is already sourced.
```

Comment:
I think we should just change everything to `docker-compose up -d` and then we don't have to worry about the confusion.